models/childModel.js

```diff
@@ -0,0 +1,35 @@
+import mongoose from 'mongoose';
+
+const guideSchema = new mongoose.Schema({
+    question: {
+        type: String,
+        required: true
+    },
+    answer: {
+        type: String,
+        required: true
+    },
+    category: {
+        type: String,
+        required: true
+    }
+});
+
+const childSchema = new mongoose.Schema({
+    name: {
+        type: String,
+        required: true
+    },
```

Comment:
-  in ths childSchema, why are you adding a property name?  is for what use?

Reply:
it was the name of the baby but if not necessary I'm going to change it also